.github/workflows/publish_ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Publish to GitHub
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  CONFIGURATION: Release
+
+jobs:
+  build:
+    # Sets permissions of the GITHUB_TOKEN to allow release creating
+    permissions:
+      packages: write
+    environment:
+      name: nuget
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup necessary dotnet SDKs
+        uses: actions/setup-dotnet@v4
+        with:
+          global-json-file: global.json
+
+      - name: Add the GitHub source
+        run: dotnet nuget add source --name "github.com" "https://nuget.pkg.github.com/fsprojects/index.json"
+
+      - name: Ensure NuGet package source mapping
```

Comment:
@xperiandri what is this exactly? I would prefer a script instead of so much githubActions logic

Reply:
moved to `build.fs`